tests/font_tests.ts

```diff
@@ -46,29 +46,24 @@ function setFont(assert: Assert): void {
   assert.equal(ctx.font, 'italic 100px PetalumaScript');
 
   const voice = new Voice();
-  // Many elements do not override the default Element.TEXT_FONT.
-  assert.propEqual(voice.fontInfo, Element.TEXT_FONT);
   voice.setFont('bold 32pt Arial');
   const fontInfo = voice.fontInfo;
   assert.equal(fontInfo?.size, '32pt');
 
   const flat = new Accidental('b');
-  // eslint-disable-next-line
-  // @ts-ignore access a protected member for testing purposes.
-  assert.equal(flat.textFont, undefined); // The internal instance variable .textFont is undefined by default.
   // Add italic to the default font as defined in Element.TEXT_FONT (since Accidental does not override TEXT_FONT).
-  flat.setFont(undefined, undefined, undefined, FontStyle.ITALIC);
-  assert.equal(flat.getFont(), 'italic 10pt Arial, sans-serif');
+  flat.setFont(undefined, undefined, undefined, 'italic');
+  assert.equal(flat.getFont(), 'italic 30pt Bravura,Gonville,Custom');
   // Anything that is not set will be reset to the defaults.
-  flat.setFont(undefined, undefined, FontWeight.BOLD, undefined);
-  assert.equal(flat.getFont(), 'bold 10pt Arial, sans-serif');
-  flat.setFont(undefined, undefined, FontWeight.BOLD, FontStyle.ITALIC);
-  assert.equal(flat.getFont(), 'italic bold 10pt Arial, sans-serif');
-  flat.setFont(undefined, undefined, FontWeight.BOLD, 'oblique');
-  assert.equal(flat.getFont(), 'oblique bold 10pt Arial, sans-serif');
+  flat.setFont(undefined, undefined, 'bold', undefined);
+  assert.equal(flat.getFont(), 'bold 30pt Bravura,Gonville,Custom');
+  flat.setFont(undefined, undefined, 'bold', 'italic');
+  assert.equal(flat.getFont(), 'italic bold 30pt Bravura,Gonville,Custom');
+  flat.setFont(undefined, undefined, 'bold', 'oblique');
```

Comment:
Nice. So we now can specify music fonts as CSS font strings.

What about the text-like elements. Do we have any tests that verify `'10pt Arial, sans-serif'` still works and parses correctly? I don't see why it wouldn't, since it's all strings, but we might as well keep the test for Arial around, in case any of our elements uses Arial as their main text font.

Reply:
There is no check on the font famly, any combination is fine. You will see that in the HACK in flow.ts in the next PR.